website_product_filters/static/src/js/test_attributes.js

```diff
@@ -24,11 +24,10 @@
             );
         }
         if (steps[k].title === "open customize menu bis") {
-            steps.splice(k, 1, {
-                title: 'Open Customize Menu',
-                element: '#customize-menu-button',
-            }
-            );
+            steps.splice(k, 1);
+        }
+        if (steps[k].title === "click on 'Product Attribute's Filters'") {
+            steps.splice(k, 1);
```

Comment:
dude, can you chill, this is a step replacement, since some other tests inherit the test I cant replace the step with its array index, the only way to do it is with the name, if your have any better idea please let me know.